README.md

```diff
@@ -71,14 +61,18 @@ Wine does not includes DXVK-NVAPI.
 
 DXVK 1.10 and older does not support `DXVK_ENABLE_NVAPI`. Disable the `nvapiHack` in DXVK 1.10 and older with `dxgi.nvapiHack = False` set in a DXVK configuration file, see [dxvk.conf](https://github.com/doitsujin/dxvk/blob/master/dxvk.conf).
 
+### Non-NVIDIA GPUs
```

Comment:
Perhaps we could refactor this section to begin with `### Using DXVK-NVAPI outside/without NVIDIA proprietary driver` and then add a subsection that contains additional info about non-NVIDIA GPUs. This way we can mention what would be needed to use DXVK-NVAPI with NVK (allow other drivers + optionally driver version) and then how to spoof NV with other vendors on top of that.